imports/collections/onlineusers_private.js

```diff
@@ -0,0 +1,38 @@
+import { Meteor } from 'meteor/meteor';
+import { OnlineUsersSchema } from './onlineusers.schema';
+import moment from 'moment/moment';
+
+if (Meteor.isServer) {
+    Meteor.publish('onlineUsersForRoute', function (route) {
+        return OnlineUsersSchema.find({activeRoute: route});
+    });
+}
+
+const checkRouteParamAndAuthorization = (route, userId) => {
+    check(route, String);
+    if (!userId) {
+        throw new Meteor.Error('not-authorized');
+    }
+};
+
+Meteor.methods({
+    'onlineUsers.enterRoute'(route) {
+        const userId = Meteor.userId();
+        checkRouteParamAndAuthorization(route, userId);
+
+        OnlineUsersSchema.insert({
```

Comment:
@migerh the user could have opened two minutes in two different browser windows (next to each other). In `isUserRemotelyConnected()` we check `userid` and the `activeRoute` properties. We could make an update if we have an entry for the user on the same route. This would make definitely sense!